src/HW1.java

```diff
@@ -0,0 +1,46 @@
+public class HW1 {
```

Comment:
better use full name, for example FirstHomework
https://mate-academy.github.io/style-guides/java/java.html#s5-naming

Reply:
why didn't you change package name?